### PR TITLE
[BEACON-373] First version

### PR DESCRIPTION
**Pull request by williamrijksen**
CommonJS module to set a capability alert, to use it for example to trigger a user to update the application or enable the Bluetooth functionality on a phone.

Check the readme.MD for the features.

### CODE REVIEW

**Comment by williamrijksen** [comment]
@jvandijk Ready for review.

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            module: {
+                src: '.',
+                dest: 'dist'
+            }
+        },
+
+        unzip: {
+            module: {
+                src: 'dist/com.sensimity.ti.capabilityalert-commonjs-<%= pkg.version %>.zip',
+                dest: '../../Koophetlokaal/Koophetlokaal-app/'
```
Comment (jvandijk) [inline Gruntfile.js]: Please remove this hardcoded reference.

File: dist/modules/commonjs/com.sensimity.ti.capabilityalert/0.1.0/com.sensimity.ti.capabilityalert.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            never: false,
+            tries: 0,
+            last: 0
+        });
+
+    if (checks.never === true) {
+        return;
+    }
+
+    if (now - checks.last < (options.tries * options.remindAfter)) {
```
Comment (jvandijk) [inline dist/modules/commonjs/com.sensimity.ti.capabilityalert/0.1.0/com.sensimity.ti.capabilityalert.js]: Hmmm... Your reminding the user not after 1, but after 3 days by doing this calculation. That's not expected when in line with the documentation

**Comment by williamrijksen** [comment]
Working on it.

**Comment by williamrijksen** [comment]
@jvandijk Ready for review. Excuse for the number of line code changes.

File: dist/modules/commonjs/com.sensimity.ti.capabilityalert/0.1.0/com.sensimity.ti.capabilityalert.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    var now = (new Date() / 1000),
+        checks = Ti.App.Properties.getObject(options.key, {
+            never: false,
+            tries: 0,
+            last: 0
+        });
+
+    if (checks.never === true) {
+        return;
+    } else if (now - checks.last < (options.tries * options.remindAfter)) {
```
Comment (jvandijk) [inline dist/modules/commonjs/com.sensimity.ti.capabilityalert/0.1.0/com.sensimity.ti.capabilityalert.js]: Huh? We just discussed this.... remove the options.tries here!

**Comment by jvandijk** [comment]
Merging!

**Pull request merged**